pyenvbuilder/commands/unpack.py

```diff
@@ -0,0 +1,52 @@
+'''
+Class for the pack-command
+'''
+import logging
+from string import Template
+from .interface import Command
+from ..utils import validate_path, run_subprocess
+
+
+logger = logging.getLogger(__name__)
+
+
+class Unpack(Command):
+    def __init__(self):
+        self.name = 'unpack'
+        self.help = 'Unpacks an environment from a tarball'
+
+    def run(self, **kwargs):
+        envs = kwargs.get('tarballs')
+        for e in envs:
+            if validate_path(e)[0]:
+                self.unpack(e)
+            else:
+                logger.error(f'When unpacking could not validate {e}')
+
+    def add_args(self, cmd_parser):
+        cmd_parser.add_argument(
+            'tarballs', nargs='+')
+
+    def unpack(self, env):
+        '''
+        Unpacks an environment from a tarball
+
+        Parameters
+        ----------
+        env: string
+            Environment.gz name/path
+        '''
+        env = env.split('.tar.gz')[0]
+
+        unpack_template = Template(
+            'mkdir $env_name\n'
+            'tar -xzf $env_name.tar.gz -C $env_name\n'
```

Comment:
I would just suggest to maybe add a `echo Unpacking the environment tarball...` so users have some feedback here.

Reply:
Yeah this is a good idea, thank you. I added it now